### DISCUSSION TITLE
Image validation is requiring an image be present

### DISCUSSION THREAD

**Discussion by kupe517** [statamic/cms#8014] [Troubleshooting] [answered]
I have an image field which I am I trying to limit the valid mime types to be either jpg, jpeg, or png. From the Statamic backend I navigated to the Validation tab and added the rule `mimes:jpg,jpeg,png`. The validation for the file formats works as expected, but the issue I am facing is that Statamic seems to think this field is required now. The `required` toggle is off.  When I try to save the page with that field empty it throws an error that the image must be of mime of type jpg,jpeg,png. How do I keep the mime type validation in place without the image being required?

**Comment by ryanmitchell** [comment]
If you add the `nullable` validation rule it should allow to to pass through when blank

**Comment by kupe517** [comment]
@ryanmitchell I tried adding the `nullable` validation in addition and I am still getting a validation error. I think the issue that the value returns an empty string '', not null.

**Comment by jasonvarga** [comment]
Where are you submitting the form? An entry publish form in the CP? A front-end form using the Forms feature? Something custom?

**Reply by kupe517** [reply]
@jasonvarga From the Statamic CP. Its an asset field that I want to be optional but also prevent PDF uploads on it.

**Reply by jasonvarga** [reply]
In that case I can't reproduce it. If I add a `mimes` rule, and leave the field as non-required, I can submit the page just fine without picking any assets.

Can you share the yaml for that field?

**Comment by kupe517** [comment, marked as answer]
It turned out that this issue is present on older versions of Statamic. I was on 3.0.38. I'm not sure at what point it was fixed but I can confirm the latest version does not have this issue.
